test/testcafe/spec/EnduserEmailConsentView_spec.js

```diff
@@ -34,13 +34,13 @@ async function setup(t) {
 test
   .requestHooks(enduserEmailConsentSuccess)('has the right title, info and button texts', async t => {
     const consentPage = await setup(t);
-    await checkA11y(t);
```

Comment:
Was removing the `checkA11y` intentional?

Reply:
Not intentional. 😬  Good catch!